kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Worker.java

```diff
@@ -131,28 +131,23 @@ protected boolean receive(Envelope envelope) {
                 })
             .collect(toList());
 
-    UUID commitId = ((CommitRequestPayload) event.payload()).commitId();
+    UUID commitId = ((StartCommit) event.payload()).commitId();
 
     List<Event> events =
         writeResults.stream()
             .map(
                 writeResult ->
                     new Event(
                         config.controlGroupId(),
-                        EventType.COMMIT_RESPONSE,
-                        new CommitResponsePayload(
+                        new DataWritten(
                             writeResult.partitionStruct(),
                             commitId,
-                            TableName.of(writeResult.tableIdentifier()),
+                            TableReference.of(config.catalogName(), writeResult.tableIdentifier()),
```

Comment:
Not sure if this is right.  `TableReference` now needs a `catalog` of type String.  I'm assuming `config.catalogName()` and `catalog.name()` are interchangeable in the usage here but I really need to test in an e2e test. 

Note: you cannot use TableReference as a comparble entity like a TableIdentifier due to having more stuff on the object that is different (for the same TableId)